### PR TITLE
2.1 version

### PR DESCRIPTION
**Pull request by abretaud**
A big PR for some code I had waiting on my hdd!
The changes are:
- auto reflect db schema (it adds a little delay at startup, but makes life much easier to work with relations between tables)
- add phylogeny module, to use with https://github.com/legumeinfo/tripal_phylotree
- make chakin util commands work when db is offline
- add minimal unit tests

I have some other things to finish before merging, I'll update this PR

### CODE REVIEW

**Comment by hexylena** [comment]
*AND* test cases? You're amazing!

File: chado/feature/__init__.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        :type organism: int
+        :param organism: Organism ID
+
+        :type sequence_type: str
+        :param sequence_type: Sequence type
+
+        :rtype: None
+        :return: None
+        """
+        raise Exception("Not implemented")
```
Comment (hexylena) [inline chado/feature/__init__.py]: `raise NotImplementedError()`?

Comment (abretaud) [reply chado/feature/__init__.py]: Ah yes, prettier :)

**Comment by abretaud** [comment]
Friday commit!
Now we have proper load_fasta and load_gff command (with tests!). I wrote it because I found the tripal loader too slow (and getting slower and slower with the size of the db).

I just tested it with loading 4 ~500Mb genomes and their annotations: with tripal it took ~4days, with python-chado it was done in <4h (I still have to rerun the benchmark though, I *think* it will even be faster now with the latest changes). (but it uses more memory, around 2Gb at the peak)

I'm nearly done with this PR, I'll just try to add more tests next week for the phylogeny part, maybe exports too.

@erasche I hope I didn't break anything with all theses changes, if you already use python-chado it would be good to check if it still works for you.

**Comment by hexylena** [comment]
WOW what a massive change. That's incredible. We're not using it here yet, so I'm happy for you to merge these whenever you're happy. My old job, if it's used there, it should be version pinned so no worries.

I can't imagine anything breaking since the public interfaces stayed more or less the same.

**Comment by abretaud** [comment]
Time for a release!

**Pull request merged**